03-use-cases/01-finance/retail-banking/main.ipynb

```diff
@@ -0,0 +1,1931 @@
+{
```

Comment:
<p>Please add explanations for why you're doing each of the below steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/824/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
This is done!